src/auth/screens/login.screen.js

```diff
@@ -153,7 +153,7 @@ class Login extends Component {
 
   signIn = () =>
     openURLInView(
-      `https://github.com/login/oauth/authorize?response_type=token&client_id=${CLIENT_ID}&redirect_uri=gitpoint://welcome&scope=user%20repo&state=${stateRandom}`
+      `https://github.com/login/oauth/authorize?response_type=token&client_id=${CLIENT_ID}&redirect_uri=gitpoint://welcome&scope=user%20repo%20read:org&state=${stateRandom}`
```

Comment:
Looks like this adds read access to org data. Without this, does your queries not work?

Reply:
Yes, without this, not getting an organization, there will be an error. Therefore, I write that there is a problem: we need to update the token when changing the scope, as for example in this case. And the question: how to do this?